generator/source/faq/docker.rst

```diff
@@ -221,6 +221,15 @@ I get this error running Docker: ``ERROR: manifest for hyperledger/sawtooth-vali
 -----------------------------------------------------------------------------------------------------
 You are following instructions for the unreleased Sawtooth ``nightly`` build. There are no Docker images for the nightly build. Instead use the ``latest`` build documentation at https://sawtooth.hyperledger.org/docs/core/releases/latest/app_developers_guide.html
 
+sawtooth-default-poet.yaml doesnt start the network successfully on subsequent runs ?
+-------------------------------------------------------------------------------------
+
+The root cause is the stale volume mounted, these are mounted for storing sawtooth keys in order to share between the containers. If you observe the error in subsequent runs that means these mounted volumes are left behind. The command "docker-compose down" does not remove the stale volumes mounted. To solve the problem, you can prune the volume as well and not just down the containers. For example
```

Comment:
Change "For example" to "For example the following removes containers and volumes"

Also break the line up into <= 80 characters.

Reply:
I have made the necessary changes and updated the PR .